ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_node_type.schema.yml

```diff
@@ -1324,6 +1324,9 @@ $defs:
               "$ref": "eos_designs#/$defs/node_type/keys/nodes"
               description: Define variables per node.
               type: list
+            mgmt_gateway:
+              type: str
+              description: Management interface gateway in IPv4 format.
```

Comment:
Move this block under defaults, it will be automatically injected under node_groups and nodes

In the description indicates that this takes precedence over the global knob

Reply:
Fixed!